doc/sphinx/source/recipes/recipe_cmorizers.rst

```diff
@@ -0,0 +1,94 @@
+.. _recipe_cmorizers:
+
+Cmorizers recipes
+=================
+
+Overview
+--------
+
+These are cmorizer recipes calling cmorizer diagnostic scripts.
+
+ESMValCore supports ERA5 hourly and montly datasets in their native
```

Comment:
Add a link to https://confluence.ecmwf.int/display/CKB/ERA5%3A+data+documentation here somewhere?

Reply:
Added.